### DISCUSSION TITLE
Sfogliare vari sheet di una plancia con movimento laterale su tablet

### DISCUSSION THREAD

**Discussion by mriitan** [indomus/forum#1482] [HUB: Home Assistant]
ciao, 
ho visto, qualche tempo fa, un video di un tizio che sfogliava i vari sheet di una plancia su un tablet a muro semplicemente scorrendo sul tablet, qualcuno ha idea di come i possa fare questa cosa?
grazie
m

**Comment by 50494554524F** [comment]
se hai installato hacs c'è un componente nella sezione frontend... devi cercare perché non mi ricordo come si chiama

**Reply by mriitan** [reply]
qualche aiutino sul nome?

**Reply by 50494554524F** [reply]
cerca swipe, ce ne sono tre da provare

**Reply by mriitan** [reply]
eccolo:
Lovelace Swipe Navigation

grazie mille
